__tests__/app.test.js

```diff
@@ -87,12 +85,33 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.msg).toBe('article does not exist');
         });
     });
-    test('GET:404 sends an appropriate status and error message when no id provided', () => {
+})
+
+describe('/api/articles', () => {
+    test('GET:200 responds with an array of article objects', () => {
         return request(app)
-        .get('/api/articles/')
-        .expect(404)
-        .then((response) => {
-            expect(response.body.msg).toBe('Route Not Found');
-        });
-    });
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toHaveLength(13)
+            body.articles.forEach((article) => {
+                expect(typeof article.title).toBe('string')
```

Comment:
This is an ok ish way of testing your objects. Don't forget we have plenty of jest matchers such as expect.objectContaining({}) as well.